meson.build

```diff
@@ -7,17 +7,17 @@ blas_deps = []
 if host_machine.system() == 'darwin'
```

Comment:
Can we remove the 'cc = meson.get_compiler('c')' on line 5?

Reply:
No we need it below

Reply:
Oh right, missed even further below.